tests/triangleObj_test.cpp

```diff
@@ -292,15 +262,19 @@ namespace objects
 
     TEST(TRIANGLEOBJ_METHOD, RayInsideTriangleDoesNotHit)
```

Comment:
This test is badly written. You assert on hit in a loop (ok), but then on values outside the loop, which makes me confused about what do you actually want to test.

Tabulated tests like this with vector of expected results are quite unreadable to be honest. you need to go back and fourth between the input data and the result vector.

Just remove the loop.

Reply:
I know I know :P I just added this to diable no discard warning at console beacuse it started to annoy me :P

Reply:
Ok. Please rewrite these tests later.

Reply:
I will delete all uneccesary test and rewrite them propery them after my project will be done, for practicing purpose. I know that they are really bad right now ;P